src/Shared/LogMessagePacketBase.cs

```diff
@@ -690,12 +803,6 @@ private void WriteEventToStream(BuildEventArgs buildEvent, LoggingEventType even
                 case LoggingEventType.BuildWarningEvent:
                     WriteBuildWarningEventToStream((BuildWarningEventArgs)buildEvent, translator);
                     break;
```

Comment:
Do we need to keep those? Especially `WriteBuildMessageEventToStream` and `WriteBuildWarningEventToStream` - since we are now extending the associated events and hence relying on their `WriteToStream`/`CreateFromStream` - it looks as possible source of errors that the 'base' events wouldn't be serialized with `WriteToStream`/`CreateFromStream`, but the derived would be serialied with those - those can easily get out of sync

Reply:
I tried it, and had to reverted it back as messages with `#IF TASKHOST` did not have required infrastructure code there.